mongo/options/encryptoptions.go

```diff
@@ -10,20 +10,12 @@ import (
 	"go.mongodb.org/mongo-driver/bson/primitive"
 )
 
-// QueryType describes the type of query the result of Encrypt is used for.
-type QueryType int
-
-// These constants specify valid values for QueryType
-const (
-	QueryTypeEquality QueryType = 1
-)
-
 // EncryptOptions represents options to explicitly encrypt a value.
 type EncryptOptions struct {
 	KeyID            *primitive.Binary
 	KeyAltName       *string
 	Algorithm        string
```

Comment:
Optional: Consider including constants for the valid Queryable Encryption algorithm strings.

Reply:
Added a QueryTypeEquality constant.